integ-test/build.gradle

```diff
@@ -124,21 +199,23 @@ compileTestJava {
 
 testClusters.all {
     testDistribution = 'archive'
```

Comment:
nit:  I personally like the verbose-ness of duplicating this line for each cluster.  

```
testClusters {
    integTest {
        testDistribution = 'archive'
        plugin ":opensearch-sql-plugin"
        setting "plugins.query.datasources.encryption.masterkey", "1234567812345678"
    }
    remoteCluster {
        testDistribution = 'archive'
        plugin ":opensearch-sql-plugin"
    }
    integTestWithSecurity {
        testDistribution = 'archive'
        plugin ":opensearch-sql-plugin"
    }
    remoteIntegTestWithSecurity {
        testDistribution = 'archive'
        plugin ":opensearch-sql-plugin"
    }
}
```

Reply:
Updated in a618be8f6